.NET/Microsoft.Recognizers.Text.DateTime/Turkish/Extractors/TurkishDatePeriodExtractorConfiguration.cs

```diff
@@ -311,14 +317,16 @@ public TurkishDatePeriodExtractorConfiguration(IDateTimeOptionsConfiguration con
 
         Regex IDatePeriodExtractorConfiguration.NowRegex => NowRegex;
 
+        bool IDatePeriodExtractorConfiguration.CheckBothBeforeAfter => DateTimeDefinitions.CheckBothBeforeAfter;
+
         string[] IDatePeriodExtractorConfiguration.DurationDateRestrictions => DateTimeDefinitions.DurationDateRestrictions;
 
         public bool GetFromTokenIndex(string text, out int index)
         {
             index = -1;
-            if (text.EndsWith("from"))
+            if (text.EndsWith("itibaren"))
```

Comment:
Please also extract such hardcoded terms to YAML. English has some examples of it being done.

Reply:
done